app/services/word_watcher.rb

```diff
@@ -110,4 +110,12 @@ def word_matches_for_action?(action, all_matches: false)
       false
     end
   end
+
+  def word_matches?(word)
```

Comment:
Do we need the word `word_` in that method name? Wouldn't `def matches?(word)` be enough? 🤔